server/src/core/workspace/services/workspace.service.ts

```diff
@@ -81,4 +81,35 @@ export class WorkspaceService {
 
     return workspace;
   }
+
+  async deleteWorkspace(id: string, select: any) {
+    const workspace = await this.findUnique({ where: { id }, select });
+
```

Comment:
we should throw exception using our assert() function to make sure the workspace has been found, etc.